libafl_qemu/src/calls.rs

```diff
@@ -99,6 +99,13 @@ where
         ret_addr
     };
 
+    #[cfg(any(cpu_target = "mips", cpu_target = "mips"))]
```

Comment:
Just `#[cfg(cpu_target = "mips")]` should suffice, right?

Reply:
Yes, it is sufficient 😅

Reply:
Yes, it doesn't need the `any()` part